src/display/api.js

```diff
@@ -1671,6 +1672,7 @@ class PDFPageProxy {
             resolve(textContent);
             return;
           }
+          textContent.lang ??= value.lang;
```

Comment:
Just a check question: can it happen that when `pump` is called we're immediately done (maybe if there is no text content in the document)? In that case we'll not set the `lang` attribute and if I understand correctly it'll be set to `undefined` by the line just below in the diff. However, the docstring at the top of this file mentions that it can only be `null` or a string, so if that case can actually happen we might need to avoid using `undefined`.

Reply:
Right, it's probably safer to just initialize it to `null` instead.